apps/docs/content/dotnet/how-to/customize-runtime.mdx

```diff
@@ -16,39 +16,45 @@ import Image from '/src/components/Image';
 
 import data from '@site/static/data.json';
 
+
+## Customise .NET Runtime environment
```

Comment:
I would delete this one, the main title is enough.

![image](https://github.com/user-attachments/assets/f8ec0de5-766a-42b7-8032-4f69eb0400fd)